src/serac/infrastructure/initialize.cpp

```diff
@@ -54,6 +56,12 @@ std::pair<int, int> initialize(int argc, char* argv[], MPI_Comm comm)
   // Start the profiler (no-op if not enabled)
   profiling::initialize(comm);
 
+  mfem::Hypre::Init();
+
+#ifdef MFEM_USE_SUNDIALS
```

Comment:
We should have our own config variable for this.

Reply:
We're only going to use it through MFEM.